v1/announcements.yaml

```diff
@@ -35,28 +27,7 @@ paths:
               status: '{{get_announcements_from_mcs.status}}'
               headers: '{{merge(get_announcements_from_mcs.headers, { "cache-control": options.announcement_cache_control})}}'
               body: '{{get_announcements_from_mcs.body}}'
-      x-amples:
-        - title: Retrieve announcements
-          request:
-            params:
-                domain: en.wikipedia.org
-          response:
-            status: 200
-            headers:
-              content-type: application/json
-            body:
-              announce:
-                - id: /.+/
-                  type: /.+/
-                  start_time: /.+/
-                  end_time: /.+/
-                  platforms: [ /.+/ ]
-                  text: /.+/
-                  action:
-                    title: /.+/
-                    url: /.+/
-                  caption_HTML: /.+/
-                  countries: [ /.+/ ]
+      x-monitor: false
```

Comment:
Hmm, should we still have some basic monitoring? Just status 200 & the presence of an announce property would already ensure that we didn't break this end point in MCS or RB.